test/drbgtest.c

```diff
@@ -137,6 +137,7 @@ static time_t reseed_time(EVP_RAND_CTX *drbg)
  */
 static int using_fips_rng(void)
 {
+#ifdef FIPS_MODULE
```

Comment:
This doesn't make sense to me. Isn't this only defined if we are building code which is going to be linked into fips.so? This test will never define it - which means this change results in `using_fips_rng` always returning 0 regardless of whether FIPS is in use or not.

Reply:
Hmmmm, other tests do similar and this test did before my earlier change.
I'm not seeing how it gets passed through however.

Never running the test would (which is what happened previously) would make it pass on non-stop.  So mystery solved there.

Reply:
> other tests do similar

I only see it in test/ecdsatest.c - which also looks like an incorrect usage.